plugins/nodes/src/js/pages/nodes/nodes-table/NodesTableContainer.js

```diff
@@ -55,17 +61,24 @@ class NodesTableContainer extends mixin(StoreMixin, QueryParamsMixin) {
   onNodeHealthStoreSuccess() {
     this.setState({
       filteredNodes: this.getFilteredNodes(),
-      receivedNodeHealthResponse: true
+      nodeHealthResponse: true
+    });
+  }
+
+  onStateStoreSuccess() {
+    this.setState({
+      masterRegion: CompositeState.getMasterNode().getRegionName()
     });
   }
 
   render() {
-    const { receivedNodeHealthResponse, filteredNodes } = this.state;
+    const { nodeHealthResponse, filteredNodes } = this.state;
 
     return (
       <NodesTable
         hosts={filteredNodes}
-        receivedNodeHealthResponse={receivedNodeHealthResponse}
+        nodeHealthResponse={nodeHealthResponse}
+        masterRegion={this.state.masterRegion}
```

Comment:
While fixing the prop type issue, put `masterRegion` in the above destructing

Reply:
Not really a deal breaker @ahoskins

Reply:
Still not a reason to let inconsistencies slip into the codebase.